tunnel/intra/split/example/main.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"crypto/tls"
+	"log"
+	"net"
+	"os"
+
+	"github.com/Jigsaw-Code/outline-go-tun2socks/tunnel/intra/split"
+)
+
+func main() {
+	if len(os.Args) < 2 {
```

Comment:
Consider using the `flags` module for better type safety and documentation.

Reply:
Done